shimmingtoolbox/shim/sequencer.py

```diff
@@ -435,6 +470,13 @@ def eval(self, coef):
 
             # TODO: Add units if possible
             # TODO: Add in anat space?
+            if self.opt_criteria == 'grad':
+                # Plot gradient realted results
+                self._plot_static_signal_recovery_mask(unshimmed, full_Gz, mask_full_binary)
+                self._plot_G_mask(np.gradient(unshimmed, axis=2), full_Gz, mask_full_binary, name='Gz')
+                self._plot_G_mask(np.gradient(unshimmed, axis=0), full_Gx, mask_full_binary, name='Gx')
+                self._plot_G_mask(np.gradient(unshimmed, axis=1), full_Gy, mask_full_binary, name='Gy')
```

Comment:
Gz is in which coordinate system? patient (through slice) or world coordinate (scanner coords)